src/plutosrc/live.html

```diff
@@ -33,12 +33,34 @@
             font-size: 16px;
             text-align: center;
         }
+        #aboutButton {
+            display: block;
+            margin: 20px; /* Adjust top margin as needed */
+            padding: 10px 20px;
+            background-color: #0066cc;
+            color: #fff;
+            border: none;
+            border-radius: 4px;
+            cursor: pointer;
+        }
+        #dataAnalyticsButton {
+            display: block;
+            margin: 20px;
+            padding: 10px 20px;
+            background-color: #0066cc;
+            color: #fff;
+            border: none;
+            border-radius: 4px;
+            cursor: pointer;
+        }
     </style>
 </head>
 <body>
     <div class="container">
         <h1>Park It!</h1>
         <div id="dataDisplay">Waiting for data...</div>
+        <button id="aboutButton">About</button>
```

Comment:
There should be only three buttons in a navigation bar, at the top of the website. A button for Live Data, one for Data Analytics and one for the about page. This navigation bar code should be copied and pasted into every html file and should look identical on all pages.